cmd/aks-periscope/aks-periscope.go

```diff
@@ -25,6 +25,13 @@ func main() {
 
 	clusterType := os.Getenv("CLUSTER_TYPE")
 
+	if utils.IsAzureStackCloud() {
+		err := utils.CopyCert()
```

Comment:
Should it rather be something like so that if we can't copy cert we exit? 
```go
if err := utils.CopyCert(); err != nil {
  log.Fataf("Failed to copy cert for Azure Stack Cloud environment: %v", err)
}
```